### PR TITLE
Upgrade srl controller

### PR DESCRIPTION
**Pull request by alexmasi**

### CODE REVIEW

**Comment by alexmasi** [comment]
@hellt There is a regression with this update, the topo creation now hangs if the tag of the image is not of a certain form

**Review by guoshiuan — APPROVED** [review]

**Comment by hellt** [comment]
> @hellt There is a regression with this update, the topo creation now hangs if the tag of the image is not of a certain form

What image tag are you referring to in particular?

File: manifests/controllers/srlinux/manifest.yaml [hunk 1]
```diff
@@ -394,7 +394,7 @@ spec:
         - --leader-elect
         command:
         - /manager
-        image: ghcr.io/srl-labs/srl-controller:0.3.1
+        image: ghcr.io/srl-labs/srl-controller:0.4.4
```
Comment (hellt) [inline manifests/controllers/srlinux/manifest.yaml:397]:
```suggestion
        image: ghcr.io/srl-labs/srl-controller:0.4.5
```
in 0.4.5 the version parsing function has been fixed to allow tags such as `version_X.Y.Z-buildnum`

**Comment by hellt** [comment]
was this superseded by another PR?

**Comment by alexmasi** [comment]
yes, however I'm hitting the control license parsing from tag issues again

will send the PR for review after I get it working

**Pull request closed without merging**